frontend/src/lib/components/ExportButton/ExportButton.tsx

```diff
@@ -29,7 +30,20 @@ export const ExportButton: React.FunctionComponent<ExportButtonProps & React.Ref
 
         const { actions } = exportsLogic
         const onExportClick = async (triggerExportProps: TriggerExportProps): Promise<void> => {
-            actions.startExport(triggerExportProps)
+            const timestamp = now().format('YYYY-MM-DD-hhmmss')
+            let modifiedContext: ExportContext | undefined = undefined
+
+            if (triggerExportProps.export_context && triggerExportProps.export_context.filename?.startsWith('export')) {
+                modifiedContext = {
+                    ...triggerExportProps.export_context,
+                    filename: `export-${timestamp}`,
```

Comment:
If the filename was `export_123`, we'd lose the `_123` part of it - maybe we wanna only **replace** `export` with `export-{timestamp}` instead?